app/components/FieldString/FieldStringEdit.jsx

```diff
@@ -113,11 +125,77 @@ export default class FieldStringEdit extends React.Component {
   constructor(props) {
     super(props)
 
+    this.cursor = null
+    this.dragBottomLimit = null
+    this.dragStart = {}
+    this.inputRefs = []
+    this.itemRefs = []
+    this.listContainerRef = null
+
+    this.mousePositionListener = ({clientY: currentY}) => {
```

Comment:
I understand that you're declaring `this.mousePositionListener` and `this.dragEndListener` as instance variables in the constructor so that you have a single reference to pass to `addEventListener`/`removeEventListener`, but I kinda feel it makes the constructor a bit messy. Could you perhaps declare them as class methods and then declare an instance variable that simply binds them?

```js
constructor(props) {
  this.mousePositionListener = this.handleMousePositionChange.bind(this)
}
```

Reply:
Done!